tests/integration/test_orgs_invites.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Integration tests for invitation methods implemented on orgs."""
+import github3
+
+from .helper import IntegrationHelper
+
+
+class TestInvitations(IntegrationHelper):
+
+    """Team integration tests."""
+
+    betamax_kwargs = {'match_requests_on': ['method', 'uri', 'json-body']}
+
+    def setUp(self):
+        super(TestInvitations, self).setUp()
+        self.basic_login()
+
+    def get_organization(self, name='mozillatw'):
+        """Get the organization for each test."""
+        self.token_login()
+        o = self.gh.organization(name)
+        assert isinstance(o, github3.orgs.Organization)
+        return o
+
+    def test_invites(self):
```

Comment:
We name each function after the method it is testing, e.g., `test_invitations`. Further we test methods of an object on the `Test<ObjectName>` class so this test should be moved to `TestOrganization`.